### PR TITLE
'legacy' SQLAlchemy and SQLAlchemy-Migrate are not supported

### PR DESCRIPTION
**Pull request by djmitche**
@tardyp let's get this in beta7 (since these versions don't work on beta7 anyway, I believe)

### CODE REVIEW

File: master/docs/manual/installation/requirements.rst [hunk 1]
```diff
@@ -70,14 +70,12 @@ Jinja2: http://jinja.pocoo.org/
 
 SQLAlchemy: http://www.sqlalchemy.org/
 
-  Buildbot requires SQLAlchemy version 0.8.1 or higher.
+  Buildbot requires SQLAlchemy version 0.8.7 or higher.
   SQLAlchemy allows Buildbot to build database schemas and queries for a wide variety of database systems.
 
 SQLAlchemy-Migrate: http://code.google.com/p/sqlalchemy-migrate/
```
Comment (rutsky) [inline master/docs/manual/installation/requirements.rst]: BTW, this URL doesn't work anymore

File: master/docs/manual/installation/requirements.rst [hunk 2]
```diff
@@ -70,14 +70,12 @@ Jinja2: http://jinja.pocoo.org/
 
 SQLAlchemy: http://www.sqlalchemy.org/
 
-  Buildbot requires SQLAlchemy version 0.8.1 or higher.
+  Buildbot requires SQLAlchemy version 0.8.7 or higher.
```
Comment (rutsky) [inline master/docs/manual/installation/requirements.rst]: Perhaps version should be bumped in `setup.py` too (now it's `>=0.8.0`)?

File: master/docs/relnotes/index.rst [hunk 3]
```diff
@@ -30,7 +30,6 @@ Deprecations, Removals, and Non-Compatible Changes
   See :bug:`3421` for status and to help out.
 
 * The ``buildbot`` Python dist now (finally) requires SQLAlchemy-0.8.0 or later and SQLAlchemy-Migrate-0.9.0 or later.
```
Comment (rutsky) [inline master/docs/relnotes/index.rst]: `SQLAlchemy-0.8.7`?

File: master/docs/manual/installation/requirements.rst (hunk 2 above)
Comment (djmitche) [inline master/docs/manual/installation/requirements.rst]: Oh, no, I should have written 0.8.0 here.  Good eye!

**Comment by rutsky** [comment]
LGTM

**Pull request merged**